kitchen/wazuh-chef/Dockerfile

```diff
@@ -27,6 +18,14 @@ RUN cd $HOME && \
     cd $HOME/wazuh-qa/kitchen/wazuh-chef/manager/files/ && \
     bundle install
 
+RUN git clone https://github.com/rshad/kitchen-docker && \
```

Comment:
This shouldn't be in any personal repo, please move it to a Wazuh repository.

Reply:
Hi @manuasir !

Now we have the fixes for `kitchen-docker` in this same branch.

Kr,

Rshad